src/core_modules/capture-core/components/WidgetTrackedEntityRelationship/NewTrackedEntityRelationship/NewTrackedEntityRelationship.container.js

```diff
@@ -0,0 +1,103 @@
+// @flow
+import React, { useCallback, useMemo, useState } from 'react';
+import * as ReactDOM from 'react-dom';
+import { withStyles } from '@material-ui/core';
+import { Widget } from '../../Widget';
+import { NewTrackedEntityRelationshipComponent } from './NewTrackedEntityRelationship.component';
+import { NewTEIRelationshipStatuses } from '../WidgetTrackedEntityRelationship.const';
+import type { Props } from './NewTrackedEntityRelationship.types';
+import { useFilteredRelationshipTypes } from '../hooks';
+import { LinkButton } from '../../Buttons/LinkButton.component';
+import { Breadcrumbs } from './Breadcrumbs/Breadcrumbs';
+import { useLocationQuery } from '../../../utils/routing';
+
+const styles = {
+    container: {
+        position: 'absolute',
+        top: 0,
+        left: 0,
+        right: 0,
+        bottom: 0,
+        backgroundColor: '#FAFAFA',
+    },
+    bar: {
+        color: '#494949',
+        padding: '8px',
+        display: 'inline-block',
+        fontSize: '14px',
+        borderRadius: '4px',
+        marginBottom: '10px',
+        backgroundColor: '#E9EEF4',
+    },
+    linkText: {
+        backgroundColor: 'transparent',
+        fontSize: 'inherit',
+        color: 'inherit',
+    },
+};
+
+export const NewTrackedEntityRelationshipPlain = ({
+    renderRef,
+    showDialog,
+    hideDialog,
+    relationshipTypes,
+    trackedEntityType,
+    classes,
+    ...passOnProps
+}: Props) => {
+    const [selectedRelationshipType, setSelectedRelationshipType] = useState();
+    const [creationMode, setCreationMode] = useState();
+    const { programId } = useLocationQuery();
+    const filteredRelationshipTypes = useFilteredRelationshipTypes(relationshipTypes, trackedEntityType, programId);
```

Comment:
Some structural suggestions that we can dive into more tomorrow/Monday:

- Move the filtering logic to the RelationshipTypeSelector component (will make the type selection live in it's own bucket instead of spreading it out)
- Build your own data structure in a hook (in the filtering hook or an additional hook) and use that data structure to display information in the selection component directly. We can reuse the display component when we are doing Event -> Tei relationships (and even for other types of relationships if we expand the model a bit). When selecting a type: save the constraint information that is needed for the link/register TEI part and final saving of the relationship (no need to re-find the relationship type here I feel)

But let's talk about this tomorrow/Monday since I didn't really elaborate here.